youki_integration_test/src/main.rs

```diff
@@ -0,0 +1,113 @@
+use std::path::PathBuf;
+
+mod command;
+mod support;
+
+use crate::support::cleanup_test;
+use crate::support::create_project_path;
+use crate::support::generate_uuid;
+use crate::support::initialize_test;
+use crate::support::print_test_results;
+use crate::support::test_builder;
+
+use crate::command::youki::Container;
+
+fn main() {
+    let project_path = create_project_path();
+    if initialize_test(&project_path).is_err() {
+        panic!("Can not initilize test.");
+    }
+    life_cycle_test(&project_path);
+    if cleanup_test(&project_path).is_err() {
+        panic!("Can not cleanup test.");
+    }
+
+    if initialize_test(&project_path).is_err() {
+        panic!("Can not initilize test.");
+    }
+    container_create_test(&project_path);
+    if cleanup_test(&project_path).is_err() {
+        panic!("Can not cleanup test.");
+    }
+}
+
+// This tests the entire lifecycle of the container.
+fn life_cycle_test(project_path: &PathBuf) {
```

Comment:
```suggestion
fn life_cycle_test(project_path: &Path) {
```
https://rust-lang.github.io/rust-clippy/master/index.html#ptr_arg